components/BasicAuthLogin.tsx

```diff
@@ -0,0 +1,55 @@
+'use client';
+
+import { useState } from 'react';
+import Link from 'next/link';
+import { credentialsSignIn } from '@/lib/actions';
+
+import Input from '@/components/shared/ui/Input';
+import Button from '@/components/shared/ui/Button';
+
+const BasicAuthLogin = () => {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+
+  const handleSubmit = async () => {
+    await credentialsSignIn({ email, password });
+  };
+
+  return (
+    <>
+      <h1 className="display-2-bold mb-5">Login</h1>
+      <section className="mb-5">
+        <Input
+          label="Email"
+          name="email"
+          placeholder="Enter your full name"
+          value={email}
+          onChange={(event) => setEmail(event?.target.value)}
+        />
+
+        <Input
+          label="Password"
+          name="password"
+          placeholder="Enter your password"
+          value={password}
+          onChange={(event) => setPassword(event?.target.value)}
```

Comment:
```suggestion
          label="Password"
          name="password"
          placeholder="Enter your password"
          value={password}
          type="password"
          onChange={(event) => setPassword(event?.target.value)}
```

Use this to hide the input's display